beacon-chain/rpc/beacon/validators.go

```diff
@@ -475,20 +477,22 @@ func (bs *Server) GetValidatorParticipation(
 		)
 	}
 
-	requestedState, err := bs.StateGen.StateBySlot(ctx, helpers.StartSlot(requestedEpoch+1))
+	nextEpochEndSlot := helpers.StartSlot(requestedEpoch+2) - 1
```

Comment:
what are these magic numbers (2 & 1)  ? Maybe we can declare them as vars here.

Reply:
It's to get `nextEpochEndSlot`, I think the var name is good enough. Forward two epochs and minus one